demo/api.md

```diff
@@ -132,18 +132,43 @@ Sets a persistent error state (e.g. an error state returned from the server). Th
 
 To give a higher limit you can bind a date to the `maxDate` property.
 
+This example demonstrates that the state of the `auro-datepicker` should reset if the date of `maxDate` is earlier than the `auro-datepicker`'s currently selected value.
+
 <div class="exampleWrapper">
-  <auro-datepicker maxDate="06/16/1980">
+  <auro-datepicker id="maxDateExample">
     <span slot="label">Choose a date</span>
-  </auro-datepicker>
+  </auro-datepicker>
+  <auro-button id="maxDateChange">Change maxDate to Today's Date</auro-button>

```

Comment:
This functional example can only be executed once without refreshing the page. We should add another button so they can reset the datepicker to the state it is in when first loaded.